src/modules-css/book.css

```diff
@@ -103,6 +105,7 @@
 
 .main-ul {
   list-style-type: none;
+  margin-left: 25px;
```

Comment:
Your design looks great. Kindly make it better and closer to the design by adjusting the following:

- The shade of the blue colors to match the used color for the page title, progress bars, add book button and update button.
- The remove button has no background and borders
- A uniform margin is expected between the main container and the book cards
- The title of the page is left aligned with the book cards
- There's no horizontal rule between the cards and the margin between the cards are bigger in your design than they appear on the design guide.
- Between the Books and Categories, the current page has a different darker color
- The category for each book appears above the name

Your design:
![image](https://user-images.githubusercontent.com/20554583/202600248-4f4290e6-ce52-4043-bb70-b75c94a83ca5.png)

Design guideline:
![image](https://user-images.githubusercontent.com/20554583/202600989-bfd7cff1-6d0b-4e97-ad69-bf0a63a170a9.png)

Reply:
Thank you for the review! I resolve all the issues, so now I must be approved :)
PD: I have some electriciry issues, so please approve me :(
![image](https://user-images.githubusercontent.com/108556234/202855695-8b9d123b-a15e-484d-8c49-67e9699fc46d.png)